src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationRun.cs

```diff
@@ -116,9 +120,9 @@ public virtual ParameterIdentificationRunResult Run(CancellationToken cancellati
          {
             RunResult.Status = RunStatus.Running;
             initialize();
-            var parameterConstraints = retrieveParameterConstraints();
 
-            RunResult.Properties = _optimizationAlgorithm.Optimize(parameterConstraints, performRun);
+            var variableParameterConstraints = retrieveVariableParameterConstraints();
+            RunResult.Properties = _optimizationAlgorithm.Optimize(variableParameterConstraints, performRun);
```

Comment:
The optimizer only 'sees' variables parameters. Fixed parameter are never passed through